### PR TITLE
Task 1/2/3/4 complete final update

### PR DESCRIPTION
**Pull request by HrisikeshSinghaChowdhury**
After Jquery in show and add blogs

### CODE REVIEW

File: app/views/microposts/create.js.erb [hunk 1]
```diff
@@ -1,3 +1,6 @@
-alert("<%='Hurray Blog is created..'%>");
-var newblogs = document.querySelector("#blog-list");
-newblogs.insertAdjacentHTML("beforeend", "<%= j render 'micropost', micropost: @micropost %>");
+$.ajax({ success: function(result){
```
Comment (Ramyani-QI) [inline app/views/microposts/create.js.erb]: @HrisikeshSinghaChowdhury why are you defining all these statements inside this ajax function? 
$.ajax is used for manually sending ajax request and handle the response.

File: app/views/microposts/new.js.erb [hunk 2]
```diff
@@ -1,7 +1,4 @@
-<% if logged_in? %>
-alert("Hii please proceed..");
-<%else%>
-alert("Sorry something went wrong");
-<%end%>
-var blogs = document.querySelector("#blogs");
-blogs.insertAdjacentHTML("beforeend", "<%= j render 'form', micropost: @micropost %>");
+$(document).ready(function(){
```
Comment (Ramyani-QI) [inline app/views/microposts/new.js.erb:1]: @HrisikeshSinghaChowdhury why do we need $(document).ready here?

File: app/views/microposts/create.js.erb (hunk 1 above)
Comment (HrisikeshSinghaChowdhury) [reply app/views/microposts/create.js.erb]: on getting successful ajax response i hide the add new section ....so i used it

File: app/views/microposts/new.js.erb (hunk 2 above)
Comment (HrisikeshSinghaChowdhury) [reply app/views/microposts/new.js.erb:1]: Ok i got it

File: app/views/microposts/create.js.erb (hunk 1 above)
Comment (Ramyani-QI) [reply app/views/microposts/create.js.erb]: https://www.w3schools.com/jquery/ajax_ajax.asp

$.ajax({url: "demo_test.txt", success: function(result){
    $("#div1").html(result);
  }});

This is the syntax and this is used when manually sending the ajax request for that given url. For your case this is not the case. we are not sending the ajax through jquery ajax function, it is handled by rails through data-remote option.

Comment (HrisikeshSinghaChowdhury) [reply app/views/microposts/create.js.erb]: Ok I understood.....

**Pull request merged**